doc/devel/documenting_mpl.rst

```diff
@@ -54,9 +54,36 @@ requirements that are needed to build the documentation. They are listed in
 
 .. note::
 
-  * You'll need a minimal working LaTeX distribution for many examples to run.
+  * You will need a minimal working LaTeX and Graphviz distribution for many
+    examples to run.
   * `Graphviz <http://www.graphviz.org/Download.php>`_ is not a Python package,
     and needs to be installed separately.
+  * Create a seprate environment using conda/venv.
+  
+  .. code-block:: sh
+
+   virtualenv venv
```

Comment:
don't use virtualenv, on any python supported by matplotlib there's python -mvenv which is builtin and does the same thing